core/objects.go

```diff
@@ -144,6 +144,9 @@ type ValidationRecord struct {
 	//   ...
 	// }
 	AddressesTried []net.IP `json:"addressesTried,omitempty"`
+	// ResolverAddress is the host:port of the DNS resolver that fulfilled the
+	// lookup for AddressUsed.
```

Comment:
This comment is not wholly true, sometimes it's `A:host:port,AAAA:host:port` which is its own bespoke format.

Reply:
What do you think I should do in this case? I considered turning `ResolverAddress` into `[]ResolverAddresses`.

Reply:
I think it's _fine_ as-is -- certainly not ideal, but no truly better solutions spring to mind -- the comment just needs to reflect it.